src/azure-cli/setup.py

```diff
@@ -140,6 +140,7 @@
     'jsondiff==1.2.0',
     'packaging~=20.9',
     'PyGithub==1.38',
```

Comment:
Could you unpin `PyGithub` (#18548, #17826)?

Reply:
Unpinned in `setup.py`. Wondering why in requirements.txt all are pinned?

Reply:
`requirements.txt` is used for building [official packages](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) like msi, deb, rpm. It is usually not possible to be used together with other python libraries in the same environment. That's why we want to pin the version. 

Also, you may check the `pip` doc on `requirements.txt`: https://pip.pypa.io/en/stable/user_guide/#requirements-files

> Requirements files are used to hold the result from pip freeze for the purpose of achieving repeatable installations. In this case, your requirement file contains a pinned version of everything that was installed when pip freeze was run.